docs/application-connector/10-04-connector-service-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Errors while generating or renewing a certificate
+type: Troubleshooting
+---
+
+## Invalid Certificate Signing Request (CSR)
+
+If you try to generate a client certificate, you may get this error:
+```json
+{
+  "code":403,
+  "error":"CSR: Invalid common name provided."
+}
+```
+
+This error is caused by applying wrong subject info to your Certificate Signing Request.  
+To ensure it was generated properly, check the values returned by the Connector Service with the call that fetched CSR info:
```

Comment:
"It" - I know you're talking about the certificate(...) but here's what I tend to do for the sake of clarity: 
1) Avoid "it" "this" and similar as much as it's humanly possible. Use the full name of "it" instead. 
2) Introduce abbreviated forms for things you need to use multiple times that have long names. Introduce the abbreviations when you use the thing's name for the first time, put it in parenthesis. Here it would be (CSR). Then you can use the abbreviated form throughout the entire document.